packages/web-app-files/src/components/SideBar/Shares/Collaborators/EditCollaborator.vue

```diff
@@ -54,6 +58,7 @@
           v-else
           id="files-collaborators-collaborator-save-share-button"
           key="edit-collaborator-saving-button"
+          data-testid="recipient-edit-btn-save"
```

Comment:
Theoretically could be removed in favor of the already exisiting `id` on that button